src/lang/en.json

```diff
@@ -119,6 +119,7 @@
       "legalTypes": {
         "CC": "BC Community Contribution Company",
         "BEN": "BC Benefit Company",
+        "CBEN": "BC Benefit Company",
```

Comment:
Also need to add CCC and CUL.

Reply:
Do you know specific translation that should be added ? Or where I can get them ?

Reply:
CCC = same as CC
CUL = same as ULC